consensus/diffs.go

```diff
@@ -27,133 +28,85 @@ type FileContractDiff struct {
 // A SiafundOutputDiff indicates the addition or removal of a SiafundOutput in
 // the consensus set.
 type SiafundOutputDiff struct {
-	New           bool
+	Direction     DiffDirection
 	ID            SiafundOutputID
 	SiafundOutput SiafundOutput
 }
 
-// A SiafundPool diff contains the value of the siafundPool before the block
+// A SiafundPoolDiff contains the value of the siafundPool before the block
 // was applied, and after the block was applied. When applying the diff, set
-// siafundPool to `Adjusted`. When reversing the diff, set siafundPool to
-// `Previous`.
+// siafundPool to 'Adjusted'. When reverting the diff, set siafundPool to
+// 'Previous'.
```

Comment:
lol. I'm not sure which I prefer, actually. \`\` makes some sense because it's what we use on github to refer to code, but '' was more prevalent in the codebase.
I think in the Go stdlib they don't use either, they just write the variable name like any other word.